search-resources-api/src/test/java/no/unit/nva/search/SearchResourcesApiHandlerTest.java

```diff
@@ -45,4 +64,24 @@ public void getSuccessStatusCodeReturnsOK() {
         assertEquals(statusCode, HttpStatus.SC_OK);
     }
 
+
```

Comment:
The naming of getSuccessStatusCodeReturnsOK makes me think that this test is tested elsewhere